core/src/scss/utilities/mixins/link/_external-link.scss

```diff
@@ -1,32 +0,0 @@
-@charset "UTF-8";
-
-//
-// @external-link($external-link, $external-link-hover)
-//
-// Default external link styling, with external icon.
-//
-// $external-link - filename of external link png & svg
-// $external-link-hover - filename of external link png & svg on hover
-//
-// Style guide: Mixins.Link.external-link
-//
-@mixin external-link($external-link: external-link, $external-link-hover: external-link-hover) {
```

Comment:
Instead of completely removing the mixin, please deprecate it and patch through to the new option.

Reply:
Done